tests/Fixer/AbstractFixerTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer;
+
+use PhpCsFixer\Tests\Fixtures\Test\AbstractFixerTest\UnconfigurableFixer;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\AbstractFixer
```

Comment:
@ntzm, class does not exist, with newer PHPUnit it is crashing our builds (starting from 2.7 branch, on which we started using newer PHPUnit)
https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/builds/299588274

Reply:
Sorry, don't know why I didn't check that. I see you've fixed it. Thanks